pkg/bloomgateway/processor.go

```diff
@@ -83,6 +83,7 @@ func (p *processor) processTasksForDay(ctx context.Context, _ string, _ config.D
 
 	for _, t := range tasks {
 		FromContext(t.ctx).AddBlocksFetchTime(duration)
+		FromContext(t.ctx).AddProcessedBlocksTotal(len(tasksByBlock))
```

Comment:
Just making sure this is correct: each task has exactly `len(tasksByBlock)` blocks?

Reply:
No, `len(tasksByBlock)` is the number of all blocks from multiple multiplexed requests.

There is another field `Stats.ProcessedBlocks`, which references the amount of blocks processed for the specific request (task).

Reply:
Hm, I'm not sure I understand why we're adding `len(tasksByBlock)` for each task then

Reply:
To see how many blocks are processed in a multiplexed request.
This is interesting for the `-bloom-gateway.block-query-concurrency` setting, which defines how many blocks are processed in parallel (as part of a multiplexed request).